Documentation update

nicholebarry opened the pull request:
Update documentation since it's been awhile.

mwilensky768 commented on outputs.md:
```diff
@@ -200,28 +216,28 @@ Text file of generated bandpass solutions. The first column is the frequency cha
+   * **obsx**: float of the positive x-axis for the observation, set by dimension/2
+   * **obsy**: float of the positive y-axis for the observation, set by elements/2
+   * **zenx**: float of the positive x-axis for the zenith, set by dimension/2
+   * **zeny**: float of the positive y-axis for the zenith, set by elements/2
```
Not sure why an axis is being referred to by a number. Can these four (`obsx`, `obsy`, `zenx`, `zeny`) be clarified a bit? If unclear for now, maybe find a way of marking for clarification later since I'd prefer this didn't hold up the merge.

nicholebarry replied:
These are extents, i.e. the max value in a certain direction. Modified. Hope that clears it up.

mwilensky768 commented on outputs.md:
```diff
@@ -282,10 +306,22 @@ Text file of generated bandpass solutions. The first column is the frequency cha
-### \<obsids\>\_\<filter\>_Dirty_\<pol\>.fits / \<obsids\>\_\<filter\>\_Model_\<pol\>.fits / \<obsids\>\_\<filter\>\_Residual_\<pol\>.fits <br />
+### \<obsids\>\_\<filter\>\_Dirty\_\<pol\>.fits / \<obsids\>\_\<filter\>\_Model_\<pol\>.fits / \<obsids\>\_\<filter\>\_Residual_\<pol\>.fits <br />
+
+Fits files containing the dirty, model, and residual (dirty minus model) images. These images are constrained to be real for XX and YY instrumental polarizations. Units are Jy per sterradian.
```
Since there are multiple sensible ways to make complex things real, can there be a comment about how that constraint is done (take real part of amplitude or something more sophisticated)?

mwilensky768 commented on outputs.md:
```diff
@@ -298,22 +334,58 @@ See cal.skymodel.source_list in Calibration for structure description. This outp
 
 ### \<obsids\>\_\<filter\>\_Dirty_\<pol\>.png / \<obsids\>\_\<filter\>\_Model_\<pol\>.png / \<obsids\>\_\<filter\>\_Residual_\<pol\>.png <br />
 
+Dirty, model, and residual (dirty minus model) images. These images are constrained to be real for XX and YY instrumental polarizations. Units are Jy per sterradian.
```
See comment for fits files on line 311.

mwilensky768 commented on outputs.md:
```diff
@@ -132,12 +140,20 @@ Text file of generated bandpass solutions. The first column is the frequency cha
   * **weights_arr**<br />
 
 ### \<obsids\>\_fhd_params.sav <br />
+  * **params**: Structure with arrays that contain various metadata in attributes listed below.
```
I had quite a bit of confusion when trying to outline these below. I've marked them for help. If these are obvious to someone, maybe fill them in now. Otherwise leave as is.

mwilensky768 reviewed (changes requested):
I left a few comments that I feel should be clarified now. Comments about `fhd_struct_init` can probably just be responded to without code changes. For documentation changes, if clarification is requested, I mean for it to be provided in the documentation now if it is very easy to figure out, otherwise I think it would be good to mark with `!Q` before merging.

nicholebarry commented on dictionary.md:
```diff
@@ -378,12 +378,21 @@ WARNING! Options in this section may change without notice, and should never be
+  -*Turn off/on*: 0/1
 
-**no_fits**: do not export fits files of the sky. This typically saves ~20Mb of memory for every fits file, which by default there are 16 for two polarizations. <br />
+**no_fits**: do not export any pngs (this includes images and plots calibration solutions) <br />
```
nice catch

nicholebarry commented on fhd_core/setup_metadata/fhd_struct_init_obs.pro:
```diff
@@ -8,13 +8,11 @@ FUNCTION fhd_struct_init_obs,file_path_vis,hdr,params, dimension=dimension, elem
 git,'describe',result=code_version,repo_path=rootdir('fhd'),args='--long --dirty'
 IF N_Elements(code_version) GT 0 THEN code_version=code_version[0] ELSE code_version=''
 
-calibration=fltarr(4)+1.
```
Unused keyword

nicholebarry commented:
@mwilensky768 addressed your comments, see if that is satisfactory for you 

Also saw that no_png wasn't actually passed to the cal procedure, and I've added a keyword I've been using a lot lately (save_beam_metadata_only). Useful for when you don't want to write 10Gb worth of beam pointers to disk.

mwilensky768 reviewed (commented):
Thanks for addressing my comments (and cleaning up some of my mess!). It looks like maybe there was a rebase? Was this to grab the `save_beam_metadata_only` functionality from a different branch? If so, can you just check that one more time by eye and let me know if the rebase went OK? It looks sane to me. Happy to approve if you're confident that didn't break anything.

nicholebarry commented:
I rebased since documentation sometimes gets put in master. Nothing more than just trying to get the full picture. 

Nothing broke.
